scripts/build/buildScripts/runWebpack.ts

```diff
@@ -18,6 +18,7 @@ export default function runWebpack(config: webpack.Configuration) {
         }
 
         if (stats?.hasErrors()) {
+          console.error(info?.errors);
```

Comment:
Was this meant to stay?

Reply:
No, it's a larger problem with the build system that I need to fix in a different PR.